assets/js/modules/search-console/components/settings/SettingsForm.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * SettingsForm component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../datastore/constants';
+import { isValidPropertyID } from '../../util';
+import ErrorText from '../../../../components/ErrorText';
+import { PropertySelect } from '../common/';
+import StoreErrorNotices from '../../../../components/StoreErrorNotices';
+const { useSelect } = Data;
+
+export default function SettingsForm() {
+	const propertyID = useSelect( ( select ) => select( STORE_NAME ).getPropertyID() );
+
+	return (
+		<div className="googlesitekit-search-console-settings-fields">
+			<StoreErrorNotices moduleSlug="search-console" storeName={ STORE_NAME } />
+
+			<div className="googlesitekit-setup-module__inputs">
+				<PropertySelect />
+			</div>
+
+			{ ( propertyID && ! isValidPropertyID( propertyID ) ) && (
+				<ErrorText message={ __( 'Not a valid property ID.', 'google-site-kit' ) } />
+			) }
```

Comment:
I think this isn't necessary – we don't have this kind of messaging in other modules for values provided by our selects so it shouldn't be possible to select an invalid value. The checks we have in place for submit changes validation should be sufficient for defensive purposes (if it was set to an invalid value programmatically for example). Unless you can think of another case for keeping this, let's remove it 🙂

Reply:
Removed.